docs/10_using_dandi.md

```diff
@@ -1,126 +0,0 @@
-   Dandiset. Click the download icon <img
-src="../img/download_file_icon.jpg"
-alt="download_file_icon"/> to download a 
-   specific file. 
-**Note:** To download an entire Dandiset, you will need to follow the instructions in the 
-   [Download](./12_download.md) section to install and use the DANDI Python client tool.
-
-### Next steps
-Although anyone on the Internet 
-can view and download public Dandisets, registered users can also create Dandisets, upload data, and publish 
-the Dandiset to generate a DOI for it.
-See the sections that follow for more detailed information about the DANDI project, as well as instructions on how 
-to work with public Dandisets or to create and publish you own as a registered user. 
-
-## Dandiset Actions
-
-The DANDI project contains the DANDI Web application, the DANDI Python client tool, and the DANDI JupyterHub 
-instance. These tools can be used to perform actions on Dandisets. 
-
-
-<img
-src="../img/dandiset_activity.svg"
-alt="dandiset_activity"
-style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
-
-
-You can learn more about the Dandiset actions in separate sections:
-
-- [View](./11_view.md)
-- [Download](./12_download.md)
-- [Upload](./13_upload.md)
-- [Publish](./14_publish.md)
-
-
-## Tools to interact with DANDI
-
-### **DANDI Web application**
-
-The [DANDI Web application](https://dandiarchive.org/) allows you to:
-
-* Search across all public Dandisets
-* Download data from public Dandisets
-* Create a new Dandiset and provide metadata
-* Publish your Dandiset
-
-
-### **DANDI Python client**
-
-The [DANDI Python client](https://pypi.org/project/dandi/) allows you to:
-
-* Download Dandisets and individual subject folders or files
-* Organize your data locally before upload
-* Upload Dandisets
-
-Before you can use the DANDI Python client, you have to install the package with `pip install dandi` in a Python 3.8+
-environment.
-
-You should check the [Dandi Debugging section](./15_debugging.md) in case of any problems.
-
-### **Dandihub analysis platform**
-
-[Dandihub](https://hub.dandiarchive.org) provides a JupyterHub instance in the cloud to interact with the data stored in DANDI.
-
-To use the hub, you will need to register for an
-account using the [DANDI Web application](https://dandiarchive.org/). 
-Note that `Dandihub` is not intended for significant computation, but provides a place to introspect Dandisets and to perform some analysis and visualization of data.
-
-## Technical limitations
-
-- **File name/path:** There is a limit of 512 characters for the full path length within a dandiset.
-- **Volume and size:** There is a limit of 5TB per file. We currently
-  accept any size of standardized datasets, as long as you can upload them over
-  an HTTPS connection. However, we ask you contact us if you plan to upload more than 10TB of data.
```

Comment:
I might have missed this, but do we want to keep this section anywhere?

Reply:
added back in